handlers/deck_handler.py

```diff
@@ -0,0 +1,74 @@
+from hearthstone import deckstrings
+import binascii
+import requests
+import json
+
+class DeckHandler():
+
+    def __init__(self, deck_response):
+        self.deck_response = deck_response
+        self.decks = self.get_decks()
+
+    def handle(self, input):
+        # Handling the message being just the deck string
+        if ' ' not in input:
+            if self.is_valid(input):
+                url = self.get_url(input)
+                if url != '':
+                    return self.deck_response + url
+        # Handling multiple words and multiple deck strings in the message.
+        result = ''
+        for word in input.split():
+            if self.is_valid(word):
+                result += self.get_url(word)
+        if result != '':
+            return self.deck_response + result
+        return None
+
+
+    @staticmethod
+    def is_valid(potentialstring):

```

Comment:
If this returns the deck or None instead, `get_url` wouldn't have to parse it a second time.